settings.gradle.kts

```diff
@@ -1,5 +1,15 @@
 @file:Suppress("SpellCheckingInspection")
+pluginManagement {
+    repositories {
+        mavenLocal()
+        gradlePluginPortal()
+    }
 
+    /** TODO: Update to Gradle 5.6.2 then use the boilerplate from here
```

Comment:
Is there a reason you're not targeting the `develop` branch which is way ahead? It's already using Gradle 5.6.2